01/implement-most-frequent-k-elements.instructor.md

```diff
@@ -0,0 +1,29 @@
+# Instructor: Implement Most Frequent K Elements
+
+An example of a working implementation:
```

Comment:
Maybe include time and space complexity? This looks like quadratic O(n^2) time and linear O(n) space to me.